src/contractIntegration/interaction/web3Provider/nonceManager/NonceManager.ts

```diff
@@ -0,0 +1,28 @@
+import 'reflect-metadata'
+import ICacheEngine from "./ICacheEngine";
+import { container } from "tsyringe";
+import MemoryCacheEngine from "./MemoryCacheEngine";
+import LocalStorageCacheEngine from "./LocalStorageCacheEngine";
+
+container.register("cache", MemoryCacheEngine);
```

Comment:
Hello @HectorGerardoZV please rename this to "memory"

Reply:
Done